recipes/physical_validation/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "physical_validation" %}
+{% set version = "1.0rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shirtsgroup/physical_validation/archive/{{ version }}.tar.gz
+  sha256: 8b7b80bbe22e555a290283e0eaa1edd6a20440f7f5e5f96887d7a51ac3c64dc5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pymbar
+
+#test:
+#  # Some packages might need a `test/commands` key to check CLI.
+#  # List all the packages/modules that `run_test.py` imports.
+#  imports:
+#    - simplejson
+#    - simplejson.tests
+
+about:
+  home: https://github.com/shirtsgroup/physical_validation
+  license: LGPL-2.1
```

Comment:
To satisfy the spdx descriptors you need either `LGPL-2.1-only` or `LGPL-2.1-or-later`. See:

https://spdx.org/licenses/